src/common/hooks/useCharacter.js

```diff
@@ -44,13 +44,13 @@ export const useCharacter = (props) => {
   };
 
   const canMove = (x, y) => {
-    if (mapSpots[y] !== undefined && mapSpots[y][x] !== undefined) {
+    if (mapSpots[y][x] !== undefined) {
```

Comment:
```suggestion
   if (mapSpots[y] !== undefined && mapSpots[y][x] !== undefined) {
```
이부분 제가 테스트 해봤는데 y만 undefined될 수 있는 구간이 있어 그부분을 나가면 에러가 나더라구요
예를 들면 오른쪽 가장 하단쪽에는 x가 없으니 그부분도 조건으로 달아야 할 것 같아요!
(이부분은 수정한 파일을 제가 갖고 있으니 제가 올릴게요 참고만 해주세요!)

Reply:
네 !